insights/tests/test_fs.py

```diff
@@ -42,3 +43,28 @@ def test_exists_ensure_path_exists():
     assert os.path.exists(path)
     fs.ensure_path(path)
     assert os.path.exists(path)
+
+
+@pytest.fixture(params=['644', '775', '777'])
+def mode(request):
+    yield request.param
+
+
+@pytest.fixture
+def mode_dir(mode):
+    path = os.path.join('/tmp/insights-test-{mode}'.format(mode=mode))
+    yield path
+    fs.remove(path)
+
+
+def test_ensure_path_mode(mode, mode_dir):
+    fs.ensure_path(mode_dir, int(mode, 8))
+    assert oct(os.stat(mode_dir).st_mode)[-3:] == mode
+
+
+def test_touch_times():
+    path = os.path.join('/tmp/insights-test-touch-times')
+    fs.touch(path, (1259798405, 1259798400))
+    assert os.stat(path).st_atime == 1259798405
+    assert os.stat(path).st_mtime == 1259798400
+
```

Comment:
@psav if you remove this blank line at the end of the file you can fix the flake8 error.  You can run `flake8` check from the root directory of the insights-core project to check for these errors.